pykafka/simpleconsumer.py

```diff
@@ -760,6 +772,19 @@ def unlock_partitions(parts):
                     success_handler=_handle_success)
                 unlock_partitions(iterkeys(partition_reqs))
 
+    def _wait_for_slot_available(self):
+        """Block until at least one queue has less than `_queued_max_messages`"""
+        if all(op.message_count > self._queued_max_messages
```

Comment:
I think 
op.message_count > self._queued_max_messages
should be
op.message_count >= self._queued_max_messages